<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>File_Name_Window</class>
	<widget class="QMainWindow" name="File_Name_Window">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>1000</width>
				<height>150</height>
			</rect>
		</property>
		<property name="sizePolicy">
			<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
				<horstretch>0</horstretch>
				<verstretch>0</verstretch>
			</sizepolicy>
		</property>
		<property name="minimumSize">
			<size>
				<width>1000</width>
				<height>150</height>
			</size>
		</property>
		<property name="maximumSize">
			<size>
				<width>1000</width>
				<height>150</height>
			</size>
		</property>
		<property name="windowTitle">
			<string>Filename Editor</string>
		</property>
		<property name="windowIcon">
			<iconset>
				<normaloff>../../../Media_Resources/EWAS_Logo/EWAS_Logo_1.svg</normaloff>
				../../../Media_Resources/EWAS_Logo/EWAS_Logo_1.svg
			</iconset>
		</property>
		<property name="styleSheet">
			<string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
				background-color: rgb(244, 221, 192);
				color:rgb(0, 0, 0);
			</string>
		</property>
		<widget class="QWidget" name="filename_centralwidget">
			<layout class="QFormLayout" name="formLayout">
				<property name="horizontalSpacing">
					<number>20</number>
				</property>
				<property name="verticalSpacing">
					<number>20</number>
				</property>
				<property name="leftMargin">
					<number>15</number>
				</property>
				<property name="topMargin">
					<number>50</number>
				</property>
				<property name="rightMargin">
					<number>20</number>
				</property>
				<item row="0" column="0">
					<widget class="QLabel" name="filename_Label">
						<property name="text">
							<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;File
								Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
							</string>
						</property>
					</widget>
				</item>
				<item row="0" column="1">
					<widget class="QLineEdit" name="filename_lineEdit">
						<property name="styleSheet">
							<string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
								background-color: rgb(255, 255, 255);
							</string>
						</property>
						<property name="text">
							<string>Default_Configuration.csv</string>
						</property>
					</widget>
				</item>
				<item row="1" column="1">
					<layout class="QHBoxLayout" name="horizontalLayout">
						<property name="leftMargin">
							<number>500</number>
						</property>
						<property name="rightMargin">
							<number>0</number>
						</property>
						<item>
							<widget class="QPushButton" name="filename_OK_button">
								<property name="minimumSize">
									<size>
										<width>100</width>
										<height>0</height>
									</size>
								</property>
								<property name="maximumSize">
									<size>
										<width>100</width>
										<height>16777215</height>
									</size>
								</property>
								<property name="baseSize">
									<size>
										<width>100</width>
										<height>0</height>
									</size>
								</property>
								<property name="styleSheet">
									<string notr="true">background-color:rgb(204, 51, 0);
										color: rgb(255, 255, 255);
										font: 12pt &quot;MS Shell Dlg 2&quot;;
									</string>
								</property>
								<property name="text">
									<string>Ok</string>
								</property>
							</widget>
						</item>
						<item>
							<widget class="QPushButton" name="filename_CANCEL_button">
								<property name="minimumSize">
									<size>
										<width>100</width>
										<height>0</height>
									</size>
								</property>
								<property name="maximumSize">
									<size>
										<width>100</width>
										<height>16777215</height>
									</size>
								</property>
								<property name="baseSize">
									<size>
										<width>100</width>
										<height>0</height>
									</size>
								</property>
								<property name="styleSheet">
									<string notr="true">background-color:rgb(204, 51, 0);
										color: rgb(255, 255, 255);
										font: 12pt &quot;MS Shell Dlg 2&quot;;
									</string>
								</property>
								<property name="text">
									<string>Cancel</string>
								</property>
							</widget>
						</item>
					</layout>
				</item>
			</layout>
		</widget>
	</widget>
	<resources/>
	<connections/>
</ui>
